csharp/Platform.Collections/Lists/ListFiller.cs

```diff
@@ -20,42 +20,119 @@ public ListFiller(List<TElement> list, TReturnConstant returnConstant)
         public bool AddAndReturnTrue(TElement element) => _list.AddAndReturnTrue(element);
 
+        /// <summary>
+        /// <para>Adds a value to the list at the first index and return true.</para>
+        /// <para>Добавляет значение в список по первому индексу и возвращает true.</para>
+        /// </summary>
+        /// <param name="element"><para>Element to add.</para><para>Добавляемый элемент.</para></param>
+        /// <returns>
+        /// <para>True value in any case.</para>
+        /// <para>Значение true в любом случае.</para>
+        /// </returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public bool AddFirstAndReturnTrue(IList<TElement> elements) => _list.AddFirstAndReturnTrue(elements);
 
+        /// <summary>
+        /// <para>Adds all elements from other list to this list and returns true.</para>
+        /// <para>Добавляет все элементы из другого списка в этот список и возвращает true.</para>
+        /// </summary>
+        /// <param name="elements"><para>List of values to add.</para><para>Список значений которые необходимо добавить.</para></param>
+        /// <returns>
+        /// <para>True value in any case.</para>
+        /// <para>Значение true в любом случае.</para>
+        /// </returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public bool AddAllAndReturnTrue(IList<TElement> elements) => _list.AddAllAndReturnTrue(elements);
 
+        /// <summary>
+        /// <para>Adds values to the list skipping the first element.</para>
+        /// <para>Добавляет значения в список пропуская первый элемент.</para>
+        /// </summary>
+        /// <param name="elements"><para>The list of values to add.</para><para>Список значений которые необходимо добавить.</para></param>
+        /// <returns>
+        /// <para>True value in any case.</para>
+        /// <para>Значение true в любом случае.</para>
+        /// </returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public bool AddSkipFirstAndReturnTrue(IList<TElement> elements) => _list.AddSkipFirstAndReturnTrue(elements);
-
+        
+        /// <summary>
+        /// <para>Adds an item to the end of the list and return constant.</para>
+        /// <para>Добавляет элемент в конец списка и возвращает константу.</para>
+        /// </summary>
+        /// <param name="element"><para>Element to add.</para><para>Добавляемый элемент.</para></param>
+        /// <returns>
+        /// <para>Constant value in any case.</para>
+        /// <para>Значение константы в любом случае.</para>
+        /// </returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public TReturnConstant AddAndReturnConstant(TElement element)
         {
             _list.Add(element);
             return _returnConstant;
         }
 
+        /// <summary>
+        /// <para>Adds a value to the list at the first index and return constant.</para>
```

Comment:
`Adds a value from the first index of other list to the end of this list and return the constant.`